lib/fathead/numpy/parse.py

```diff
@@ -206,16 +213,47 @@ def get_abstract(section_div):
         rendered_returns = RETURNS_TEMPLATE.format(returns=returns)
     else:
         rendered_returns = ""
+    
+    examples = get_examples(section_div, "highlight-python")
+    if examples:
+        rendered_examples = EXAMPLES_TEMPLATE.format(examples=examples)
+    else:
+        rendered_examples = ""    
 
     abstract = ABSTRACT_TEMPLATE.format(
         rendered_information=scrub_text(rendered_information),
         rendered_signature=scrub_text(rendered_signature),
         rendered_parameters=scrub_text(rendered_parameters),
-        rendered_returns=scrub_text(rendered_returns)
+        rendered_returns=scrub_text(rendered_returns),
+        rendered_examples=scrub_text(rendered_examples)
     )
     return abstract
 
 
+def get_examples(section_div, examples_class):
+    """Parse and return the parameters or returns of the documentation topic.
+
+    Parameters
+    ----------
+    section_div : bs4.BeautifulSoup
+        The BeautifulSoup object corresponding to the div with the "class"
+        attribute equal to "section" in the html doc file.
+
+    examples_class: Str
+        The value of the "class" attribute of the <div> tag within section_div.
+        "highlight-python" is the class which are used for examples in the webpage. 
```

Comment:
Please wrap lines to 79 characters (See PEP8). So "webpage" needs to go on a new line.